publish/artifactory.gradle

```diff
@@ -1,7 +1,7 @@
 def repo = System.getenv('ARTIFACTORY_REPO')
 def user = System.getenv('ARTIFACTORY_USER')
 def password = System.getenv('ARTIFACTORY_PASSWORD')
```

Comment:
Maybe the other properties should also have the option to use Gradle properties? What do you think?

Reply:
Yes, you're right.
I applied gradle properties to other settings and also for Bintray details.
Please check.